app/Repositories/Team/TeamRepository.php

```diff
@@ -223,6 +350,24 @@ public function removeTeamProjectUser($team, $user)
         }
     }
 
+    /**
+     * Remove Team invited user
+     *
+     * @param $team
+     * @param $email
+     */
+    public function removeInvitedUser($team, $email)
+    {
+        $team = $this->model->find($team->id);
+
+        if ($team) {
+                DB::table('invited_team_users')
+                    ->where('team_id', $team->id)
+                    ->where('invited_email', $email)
+                    ->delete();
+        }
+    }
```

Comment:
Looks to be double indented.

Reply:
Removed the double indentation in TeamRepository@removeInvitedUser().